Makefile

```diff
@@ -36,8 +36,8 @@ test-with-cover:
 	 	go tool cover -html=coverage.txt -o coverage.html ); \
 	done
 
-.PHONY: go-tidy
-go-tidy:
+.PHONY: gotidy
```

Comment:
One less character to type. I like it :-)

Reply:
:) also in line with other targets like `gofmt`